solidity/contracts/system/IKeepFactorySelector.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.17;
+
+import {IBondedECDSAKeepFactory} from "@keep-network/keep-ecdsa/contracts/api/IBondedECDSAKeepFactory.sol";
+
+/// @title Keep factory selector.
+/// @notice Selects an appropriate keep factory for the given application.
+interface IKeepFactorySelector {
+
+    /// @notice Selects keep factory for the given application.
+    /// @param _application Application address.
+    /// @param _regularFactory Regular keep factory.
+    /// @param _fullyBackedFactory Fully backed keep factory.
+    /// @return Selected keep factory.
+    function selectFactory(
+        address _application,
```

Comment:
Do we need this parameter? I think the whole point is that the selection strategy is specific to tBTC, especially that this interface is specified in tBTC system contracts. I think it's fair to assume the strategy will be uniquely deployed for tBTC.

Reply:
Yeah, you're right.